sympy/series/formal.py

```diff
@@ -777,8 +777,13 @@ def _compute_fps(f, x, x0, dir, hyper, order, rational, full):
         return (result[0], result[1].subs(x, rep2 + rep2b),
                 result[2].subs(x, rep2 + rep2b))
 
+    k = Dummy('k')
     if f.is_polynomial(x):
-        return None
+        c = Symbol('c')
+        ind = f.coeff(x, 0)
+        xk = sequence(x**k, (k, 0, oo))
+        ak = sequence(c, (k, 0, oo))
```

Comment:
I suppose that `ak` should be the coefficient sequence of the polynomial `f`.

Reply:
I didn't get you, @jksuom ?

Reply:
A polynomial can be considered as a formal power series having only a finite number of non-zero coefficients. For example, the the coefficient sequence `ak` of `f = 2 + 3*x + 4*x**2 + 5*x**3` is `2, 3, 4, 5, 0, 0, 0, ...`.

Reply:
Yes, `ak` is the coefficient sequence of polynomial f, with `ind` being the constant coefficient of the polynomial..

Reply:
But this implementation makes `ak` an infinite sequence with constant value `c` instead of the finite sequence of coefficients of `f`.

Reply:
I implemented it along the lines of this idea -

`ak[:3] = [c, c, c]`

This c can be replaced by any constant present in the coefficient sequence of that particular polynomial. c can represent 1,2 or any other no. . And I extended it till infinity, since we don't know the length of the coefficient sequence. 
In this issue, we worry about the type of fps being declared as Formal Power Series, rather than a class of Pow.
Is my idea correct?
If not, what can I do else?
@jksuom

Reply:
I think that `ak` should be implemented as indicated [above](https://github.com/sympy/sympy/pull/13854#discussion_r161685158).